### DISCUSSION TITLE
iOS - Can't retrieve accounts with custom currencies

### DISCUSSION THREAD

**Discussion by sebashepin** [victorbalssa/abacus#269] [Bug] [answered]
## Environment
- Version `0.13.2`
- iPhone 13 mini
- Using a personal access token for auth

## Bug

Custom currencies, such as Colombian Peso (COP) cause the mobile app to ignore accounts which use it. If ALL accounts use the custom currency, the application hangs with a white screen

## Reproduction steps:

1. Create a custom currency with the following parameters.
![image](https://github.com/victorbalssa/abacus/assets/1766748/3b2831d5-bfdf-4fe6-9540-4d3a74de36fd)
2. Assign it to an account
3. The account is now invisible
4. Assign it to all accounts
5. Application now hangs up

**Comment by victorbalssa** [comment, marked as answer]
Hey @sebashepin 

Can you update the `Code` to: `COP` and use another symbol like `C` or `CO$` ?

It woks well on my end with these 2 changes.
![image](https://github.com/victorbalssa/abacus/assets/12813321/2eb045b8-ca2f-46f3-a884-f91f54ab3a44)

**Reply by sebashepin** [reply]
You're right, it works now. Thank you so much
